kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -871,49 +871,66 @@ object KyuubiConf {
       .intConf
       .createWithDefault(100)
 
-  val SERVER_STATE_STORE_CLASS: ConfigEntry[String] =
-    buildConf("kyuubi.server.state.store.class")
-      .doc("Fully qualified class name for server state store.")
+  val SERVER_METADATA_STORE_CLASS: ConfigEntry[String] =
```

Comment:
maybe we can remove `server` here, cc @ulysses-you @pan3793

Reply:
+1